### PR TITLE
Auto-annotate openshift resources if annotations are not present

### PR DESCRIPTION
**Pull request by aditya-konarde**

### CODE REVIEW

**Comment by aditya-konarde** [comment]
I still need to test this. 

/hold

**Comment by aditya-konarde** [comment]
I still need to test this AND add a unit test for it.

/hold

**Comment by maorfr** [comment]
This looks great! :)

**Comment by maorfr** [comment]
lets reabse and update the openshift_resouces version by a minor bump?

File: reconcile/openshift_resources.py [hunk 1]
```diff
@@ -439,15 +439,13 @@ def realize_data(dry_run, oc_map, ri, enable_deletion=True):
             c_item = data['current'].get(name)
 
             if c_item is not None:
-                # don't apply if it doesn't have annotations
+                #  If resource deoesn't have annotations, annotate and apply
```
Comment (maorfr) [inline reconcile/openshift_resources.py]: typo

**Pull request merged**